lib/blockscore/errors/invalid_request_error.rb

```diff
@@ -19,9 +19,9 @@ def initialize(response)
     end
 
     def to_s
-      status_string = @http_status ? "(Status: #{@http_status})" : ''
-      type_string = @error_type ? "(Type: #{@error_type})" : ''
-      param_string = @param ? "(#{@param})" : ''
+      status_string = @http_status ? "(Status: #{@http_status})" : ""
+      type_string = @error_type ? "(Type: #{@error_type})" : ""
+      param_string = @param ? "(#{@param})" : ""
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.